main.go

```diff
@@ -84,6 +84,9 @@ to quickly create a Cobra application.`,
 	deployCmd.Flags().StringVar(&config.Build.ContainerRegistry, "container-registry", "", "Ex. 1234.dkr.ecr.us-west-2.amazonaws.com ")
 	deployCmd.Flags().StringVar(&config.Build.ProjectId, "project-id", "", "Ex. opszero")
 	deployCmd.Flags().StringVar(&config.Build.Image, "image", "", "Ex. deploytag")
+	deployCmd.Flags().StringVar(&config.CloudFlareKey, "cloudflare-key", "", "api key for cloudflare")
```

Comment:
Just make these at the root level and have them look for CLOUDFLARE_EMAIL and CLOUDFLARE_APIKEY environment variables as a default.